lebedigital/ConcreteOntology/BaseOntologies/tax2ont/main.py

```diff
@@ -0,0 +1,126 @@
+import pandas as pd
+import shutil
+import pathlib
+import os
+
+# made by mattheo krueger (mattheo-benedikt.krueger@bam.de) for the project lebedigital
+
+# change the prefix if ontology prefix changes
+Prefix = "http://www.semanticweb.org/mkrueger/ontologies/2023/4/untitled-ontology-47/"
+# Prefix of the Annotation Property used for definition
+PrefixDef = "http://www.w3.org/ns/prov#definition"
+# Prefix of the Annotation Property used for definition source
+PrefixDefSource = "https://w3id.org/pmd/co/definitionSource"
+NameExcelFile = "Taxo.xlsx"
+
+
+# welcome message
+print("Hi, this tool is for converting a taxonomy from an Excel file to an ontology.")
+# Eingabe Aufforderung
+wahl = input('Do you want to add to a already existing Ontology (Press "1") or use a blank template (Press "2")?')
```

Comment:
so this is used "manually" in the process of building an ontology, not a function that needs to be integrated into an automated workflow?

Reply:
@eriktamsen sorry for the mess. I forgot to delete the files I used for creating the Ontology and the old BaseOntologies. I removed all unnecessary files. Also the Taxonomie is now in .csv